features/jira.py

```diff
@@ -0,0 +1,27 @@
+from datetime import date
+from jira import JIRA
+from pprint import pprint, pformat
+import settings
+
+if settings.CREATE_JIRA_VERSIONS == "True":
+    jira_options = {
+        'server': settings.JIRA_SERVER
+    }
+    jira = JIRA(options=jira_options, basic_auth=(settings.JIRA_USERNAME, settings.JIRA_PASSWORD))
+      
+def create_jira_release(change_number, user_name, description):
+    if settings.CREATE_JIRA_VERSIONS == "True":
```

Comment:
Once you have the config variable as a bool, you can do the following to test for truth

`if settings.CREATE_JIRA_VERSIONS:`